### ISSUE TITLE
When 2 methods to an interface both contain a custom data class, causes a "Conflicting import, imported name is ambiguous" build error

### ISSUE THREAD

**Issue by realityexpander** [Kotlin/kotlinx-rpc#161]
Using the following interface:

```
@Serializable
data class UserData(
    val lastName: String,
    val address: String
)

interface NewsService : RPC {
    suspend fun hello(platform: String, userData: UserData): String
    suspend fun hello2(platform: String, userData: UserData): String // Note: UserData used twice

    suspend fun subscribeToNews(): Flow<String>
    suspend fun subscribeToTopic(topic: String): Flow<String>
}
```

causes this build error:

```
Executing tasks: [desktopRun] in project /Volumes/TRS-83/dev/KotlinX-RPC-Demo/composeApp

e: file:///Volumes/TRS-83/dev/KotlinX-RPC-Demo/shared/build/generated/ksp/jvm/jvmMain/kotlin/kotlinx/rpc/NewsServiceClient.kt:12:8 Conflicting import, imported name 'UserData' is ambiguous
e: file:///Volumes/TRS-83/dev/KotlinX-RPC-Demo/shared/build/generated/ksp/jvm/jvmMain/kotlin/kotlinx/rpc/NewsServiceClient.kt:13:8 Conflicting import, imported name 'UserData' is ambiguous

FAILURE: Build failed with an exception.
```

the offending bullt file:

```
@file:Suppress("RedundantUnitReturnType", "RemoveRedundantQualifierName", "USELESS_CAST", "UNCHECKED_CAST", "ClassName", "MemberVisibilityCanBePrivate", "KotlinRedundantDiagnosticSuppress", "UnusedImport", "detekt.all")
@file:OptIn(InternalRPCApi::class)

package kotlinx.rpc

import kotlinx.coroutines.*
import kotlinx.serialization.Serializable
import kotlinx.serialization.Contextual
import kotlinx.rpc.internal.*
import kotlin.reflect.typeOf
import kotlin.coroutines.CoroutineContext
import UserData
import UserData // Note: IMPORTED TWICE
import NewsService


@Suppress("unused")
class NewsServiceClient(
    private val __rpc_stub_id: Long,
    private val __rpc_client: RPCClient,
) : NewsService {
    override val coroutineContext: CoroutineContext = __rpc_client.provideStubContext(__rpc_stub_id)

    private val __rpc_scope: CoroutineScope = this

    @Serializable
    @Suppress("unused")
    internal class Hello_RPCData(
        val platform: kotlin.String,
        val userData: UserData,
    ) : RPCMethodClassArguments {
        override fun asArray(): Array<Any?> = arrayOf(platform, userData)
    }

    override suspend fun hello(platform: kotlin.String, userData: UserData): kotlin.String = scopedClientCall(__rpc_scope) {
        val returnType = typeOf<kotlin.String>()
        val dataType = typeOf<Hello_RPCData>()
        __rpc_client.call(RPCCall("NewsService", __rpc_stub_id, "hello", RPCCall.Type.Method, Hello_RPCData(platform, userData), dataType, returnType))
    }

    @Serializable
    @Suppress("unused")
    internal class Hello2_RPCData(
        val platform: kotlin.String,
        val userData: UserData,
    ) : RPCMethodClassArguments {
        override fun asArray(): Array<Any?> = arrayOf(platform, userData)
    }

    override suspend fun hello2(platform: kotlin.String, userData: UserData): kotlin.String = scopedClientCall(__rpc_scope) {
        val returnType = typeOf<kotlin.String>()
        val dataType = typeOf<Hello2_RPCData>()
        __rpc_client.call(RPCCall("NewsService", __rpc_stub_id, "hello2", RPCCall.Type.Method, Hello2_RPCData(platform, userData), dataType, returnType))
    }

    @Serializable
    @Suppress("unused")
    internal object SubscribeToNews_RPCData : RPCMethodClassArguments {
        override fun asArray(): Array<Any?> = emptyArray()
    }

    override suspend fun subscribeToNews(): kotlinx.coroutines.flow.Flow<kotlin.String> = scopedClientCall(__rpc_scope) {
        val returnType = typeOf<kotlinx.coroutines.flow.Flow<kotlin.String>>()
        val dataType = typeOf<SubscribeToNews_RPCData>()
        __rpc_client.call(RPCCall("NewsService", __rpc_stub_id, "subscribeToNews", RPCCall.Type.Method, SubscribeToNews_RPCData, dataType, returnType))
    }

    @Serializable
    @Suppress("unused")
    internal class SubscribeToTopic_RPCData(
        val topic: kotlin.String,
    ) : RPCMethodClassArguments {
        override fun asArray(): Array<Any?> = arrayOf(topic)
    }

    override suspend fun subscribeToTopic(topic: kotlin.String): kotlinx.coroutines.flow.Flow<kotlin.String> = scopedClientCall(__rpc_scope) {
        val returnType = typeOf<kotlinx.coroutines.flow.Flow<kotlin.String>>()
        val dataType = typeOf<SubscribeToTopic_RPCData>()
        __rpc_client.call(RPCCall("NewsService", __rpc_stub_id, "subscribeToTopic", RPCCall.Type.Method, SubscribeToTopic_RPCData(topic), dataType, returnType))
    }

    @Serializable
    @Suppress("unused")
    internal object Ping_RPCData : RPCMethodClassArguments {
        override fun asArray(): Array<Any?> = emptyArray()
    }

    override suspend fun ping(): kotlin.String = scopedClientCall(__rpc_scope) {
        val returnType = typeOf<kotlin.String>()
        val dataType = typeOf<Ping_RPCData>()
        __rpc_client.call(RPCCall("NewsService", __rpc_stub_id, "ping", RPCCall.Type.Method, Ping_RPCData, dataType, returnType))
    }


    companion object : RPCClientObject<NewsService> {
        private val methodNames: Map<String, kotlin.reflect.KType> = mapOf(
            NewsService::hello.name to typeOf<NewsServiceClient.Hello_RPCData>(),
            NewsService::hello2.name to typeOf<NewsServiceClient.Hello2_RPCData>(),
            NewsService::subscribeToNews.name to typeOf<NewsServiceClient.SubscribeToNews_RPCData>(),
            NewsService::subscribeToTopic.name to typeOf<NewsServiceClient.SubscribeToTopic_RPCData>(),
            NewsService::ping.name to typeOf<NewsServiceClient.Ping_RPCData>(),
        )

        @Suppress("unused")
        override fun methodTypeOf(methodName: String): kotlin.reflect.KType? = methodNames[methodName]

        override fun withClient(serviceId: Long, client: RPCClient): NewsService = NewsServiceClient(serviceId, client)

        override fun rpcFields(service: NewsService): List<RPCDeferredField<*>> = with(service) {
            return emptyList()
        }
    }
}
```

**Comment by Mr3zee** [comment]
Hi, thank you for the report!
Will fix it soon

**Comment by Mr3zee** [comment]
Fixed in #162
